OpenRA.Mods.Common/UtilityCommands/CheckSequenceSprites.cs

```diff
@@ -39,7 +39,7 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 				Console.WriteLine("Tileset: " + ts.Name);
 				var sc = new SpriteCache(modData.DefaultFileSystem, modData.SpriteLoaders, new SheetBuilder(SheetType.Indexed));
 				var nodes = MiniYaml.Merge(modData.Manifest.Sequences.Select(s => MiniYaml.FromStream(modData.DefaultFileSystem.Open(s), s)));
-				foreach (var n in nodes.Where(node => !node.Key.StartsWith("^")))
+				foreach (var n in nodes.Where(node => !node.Key.StartsWith(SymbolicSupportDirectory.Identifier)))
```

Comment:
This looks like we're mixing directory and actor name identifiers. (?)

Reply:
That's right- actors are on those nodes. Fixed.